<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialProtocolDebugger</class>
 <widget class="QWidget" name="SerialProtocolDebugger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cb_port">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>YaHei Consolas Hybrid</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_baudrate">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>YaHei Consolas Hybrid</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>230400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>460800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>512000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>921600</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="bt_openclose">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>YaHei Consolas Hybrid</family>
    </font>
   </property>
   <property name="text">
    <string>OPEN</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="brow_t">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_protocol">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>YaHei Consolas Hybrid</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>raw</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gnss</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
